docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -126,23 +126,9 @@ dependencies {
 
 == Enable Tests
 
-Quarkus uses Junit5 and to enable it in Gradle we need to add a section to our build file:
-
-[source,groovy,subs=attributes+]
-----
-test {
-  useJUnitPlatform()
-}
-----
-
-Using the Kotlin DSL, add:
-
-[source,kotlin,subs=attributes+]
-----
-tasks.test {
-    useJUnitPlatform()
-}
-----
+When the Quarkus Gradle plugin is applied to a build, the `test` task is configured to run regular tests while the `testNative` task
+is meant to run the native image tests.
+The source code for the native image tests should reside in `src/nativeTest` instead of `src/test` where the source code for regular tests resides.
```

Comment:
```suggestion
The source code for the native image tests should reside in `src/native-test` instead of `src/test` where the source code for regular tests resides.
```

Reply:
Ah darn, I missed it, sorry!

Reply:
Fixed, thanks